{
    "CVE_data_meta": {
        "ASSIGNER": "secalert@redhat.com",
        "ID": "CVE-2019-10217",
        "STATE": "PUBLIC",
        "UPDATED": "2020-06-15"
    },
    "cna-container": {
        "affected": {
            "vendors": [
                {
                    "products": [
                        {
                            "versions": [
                                {
                                    "version_value": "ansible 2.8.0 before 2.8.4"
                                }
                            ]
                        }
                    ],
                    "vendor_name": "Red Hat"
                }
            ]
        },
        "descriptions": [
            {
                "lang": "eng",
                "value": "A flaw was found in ansible 2.8.0 before 2.8.4. Fields managing sensitive data should be set as such by no_log feature. Some of these fields in GCP modules are not set properly. service_account_contents() which is common class for all gcp modules is not setting no_log to True. Any sensitive data managed by that function would be leak as an output when running ansible playbooks."
            }
        ],
        "metrics": [
            {
                "cvss-v_3_0": {
                    "vectorString": "5.7/CVSS:3.0/AV:N/AC:L/PR:L/UI:R/S:U/C:H/I:N/A:N",
                    "version": "3.0"
                }
            }
        ],
        "problemtypes": {
            "descriptions": [
                [
                    {
                        "CWE-ID": "CWE-200",
                        "description": "CWE-200",
                        "lang": "eng",
                        "type": "CWE"
                    }
                ]
            ]
        },
        "references": [
            {
                "name": "https://bugzilla.redhat.com/show_bug.cgi?id=CVE-2019-10217",
                "refsource": "CONFIRM",
                "url": "https://bugzilla.redhat.com/show_bug.cgi?id=CVE-2019-10217"
            },
            {
                "name": "https://github.com/ansible/ansible/issues/56269",
                "refsource": "CONFIRM",
                "url": "https://github.com/ansible/ansible/issues/56269"
            },
            {
                "name": "https://github.com/ansible/ansible/pull/59427",
                "refsource": "CONFIRM",
                "url": "https://github.com/ansible/ansible/pull/59427"
            },
            {
                "name": "openSUSE-SU-2020:0513",
                "refsource": "SUSE",
                "url": "http://lists.opensuse.org/opensuse-security-announce/2020-04/msg00021.html"
            },
            {
                "name": "openSUSE-SU-2020:0523",
                "refsource": "SUSE",
                "url": "http://lists.opensuse.org/opensuse-security-announce/2020-04/msg00026.html"
            }
        ]
    },
    "data_format": "MITRE",
    "data_type": "CVE",
    "data_version": "5.0"
}